### PR TITLE
Manage plugins on webapp

### PR DESCRIPTION
**Pull request by funcionante**
This targets issue #263 and looks like the image below.
show_plugins_dicoogle
For now, plugins are expanded with its attributes when clicked, which may not be so useful, because some fields like `type` and `name` are not displayed and others are `null`. The display of this information can be better personalized when we add actions to plugins.

Depends on #310.

### CODE REVIEW

**Comment by funcionante** [comment]
Now, with the endpoints introduced in #310, it is possible to enable or disable plugins, like this:
enable_disable_plugins_webapp

File: dicoogle/src/main/resources/webapp/js/stores/pluginStore.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+const PluginStore = Reflux.createStore({
+  listenables: PluginActions,
+  init: function () {
+    this._contents = {};
+  },
+
+  onGet: function(type) {
+    const self = this;
+
+    request(Endpoints.base + "/plugins/" + type,
```
Comment (Enet4) [inline dicoogle/src/main/resources/webapp/js/stores/pluginStore.js]: Please use `dicoogle-client` directly:

```
Dicoogle.request('GET', ['plugins', type])
    .end((err, data) => { ... });
```

File: dicoogle/src/main/resources/webapp/js/components/management/pluginsView.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                            this,
+                            type,
+                            plugin.name,
+                            (plugin.enabled ? "disable" : "enable"))}>
+                    {plugin.enabled ? "Disable" : "Enable"}
+                  </button>
+                  {
+                    Object.keys(plugin).filter(field => ignoreFieldList.indexOf(field) < 0).map(field => (
+                      <p><b>{field}:</b> {
+                        plugin[field] !== null ? plugin[field].toString() : "undefined"
```
Comment (Enet4) [inline dicoogle/src/main/resources/webapp/js/components/management/pluginsView.js]: Let's hide undefined fields instead, as they do not provide useful information (and some of them are just placeholders for content not yet available).

**Comment by funcionante** [comment]
Done. I also changed the messages to be displayed in "toasts", like in [IndexerView](https://github.com/bioinformatics-ua/dicoogle/blob/f77cc78a1b12b6cd6a8635fe9e8e7ca68adbbfd8/dicoogle/src/main/resources/webapp/js/components/management/indexerView.js).

**Comment by Enet4** [comment]
I could reproduce this, although it wasn't a trivial finding in my setup:

screenshot-2018-4-26 dicoogle

If the plugin's name is very long, there's a chance that the arrow is awkwardly placed in the next line when resizing the window. It actually looks OK if we compress the window further:

screenshot-2018-4-26 dicoogle 1

Fixing this might require a bit of CSS tweaking.

**Comment by funcionante** [comment]
Well noted, I think now it is fixed.

**Pull request merged**